static/midcom.helper.datamanager2/tinymce/imagetools_functions.js

```diff
@@ -0,0 +1,46 @@
+var imagetools_functions =
+{
+    setup: function(editor) 
+    {
+        editor.on('click', function(e) 
+        {
+            var node = tinymce.activeEditor.selection.getNode();
+            if(node.hasAttribute("src"))
+            {
+                tinyMCE.settings.original = node.src.split("/").pop();
```

Comment:
Did you test if this works if there is more than one tinymce instance active on the page? Because it looks almost as if you're using `tinyMCE` as a superglobal